addons/website_sale/static/src/js/website_sale_tour_buy.js

```diff
@@ -81,8 +81,13 @@ tour.register('shop_buy_product', {
         {
             content: "finish",
             trigger: '.oe_website_sale:contains("Pending... The order will be validated after the payment.")',
+            // Leave /shop/confirmation to prevent RPC loop to /shop/payment/get_status.
+            // The RPC could be handled in python while the tour is killed (and the session), leading to crashes
+            run: function () {
+                window.location.href = '/';
```

Comment:
Are you sure that we don't have to wait for the '/' page to be fully loaded ?

Reply:
Hm, probably better yeah. Added a step to wait for the page to be loaded